packages/go_router_builder/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.2.2
+
+* Generated routes now adhere to the updated `push` method by having the ability to return the value. [go_router CHANGELOG](https://github.com/flutter/packages/blob/main/packages/go_router/CHANGELOG.md#650)
```

Comment:
I think it's more consistent with other change log if we say "Supports returning value in generated push method"

Reply:
Hey @hangyujin, thanks for the feedback. I agree. 👍